### ISSUE TITLE
Open tracking pixel is visible in some email clients

### ISSUE THREAD

**Issue by victoras** [freescout-helpdesk/freescout#499]
When the customer receives my replay, at the bottom of the mail it displays a 404 image.

<img width="1193" alt="Screenshot 2020-02-22 17 56 23" src="https://user-images.githubusercontent.com/613284/75100576-691dab80-559d-11ea-83c6-961ec296fbda.png">

<img width="666" alt="Screenshot 2020-02-22 18 04 37" src="https://user-images.githubusercontent.com/613284/75100594-cc0f4280-559d-11ea-874c-24bce925cf4c.png">

How can I fix it?

**Comment by freescout-helpdesk** [comment]
This is an "open tracking" pixel. Which email client is used to read the email?

**Comment by victoras** [comment]
I am using google suite, Gmail.

**Comment by TimoStramann** [comment]
Can confirm this error on outlook too. Many emails are automatically marked as spam by office 365 because of this I suppose. Would be great to add an option to disable this. If the email is send or converted as text-only the image turns into a row of unsorted characters. See #461 1st screenshot on how this ends.

**Comment by freescout-helpdesk** [comment]
If anybody have any ideas how to fix feel free to share.

**Comment by freescout-helpdesk** [comment]
Those who has this issue, please try to get the actuall error (is it 404 error or some other) by using browser debugger for example.

**Comment by TimoStramann** [comment]
This appears to be an issue happening with outlook, Gmail and those other big freemium services mostly. Outlooks/Gmails default security settings do not allow downloading pictures and Outlook 365 does not even allow downloading pictures at the end of messages though downloading pictures are allowed (needs another confirmation to download these pictures too). If the download of the picture is blocked, it's displayed as a question mark sign (Gmail) or empty box (outlook). Maybe having a per user setting for using the open tracking pixel would be a solution?

**Comment by TimoStramann** [comment]
Automatically detecting by the email domain and email meta/header whether it's outlook/Gmail and don't send the tracking pixel (optional setting) could be a solution to reduce the number of users getting shown this confusing symbols.

**Comment by fulldecent** [comment]
This is NOT an issue with FreeScout.

The actual issue is that tracking pixels are stupid, they are rightfully blocked by email clients, and they rightfully produce a poor experience for email recipients.

Recommending that you turn this feature off on your own installation and that this issue be closed.

---

Possibly to be filed as a separate issue, tracking pixels should be turned off by default rather than the current on by default.

**Comment by fulldecent** [comment]
Recommending to close issue. Nothing to change here. Everything is working.
